Formula/volatility.rb

```diff
@@ -160,10 +160,10 @@ def install
         s.gsub! "ZLIB_ROOT = None", "ZLIB_ROOT = ('#{sdkprefix}/usr/lib', '#{sdkprefix}/usr/include')"
         s.gsub! "JPEG_ROOT = None",
                 "JPEG_ROOT = ('#{Formula["jpeg"].opt_prefix}/lib', " \
-                             "'#{Formula["jpeg"].opt_prefix}/include')"
+                "'#{Formula["jpeg"].opt_prefix}/include')"
         s.gsub! "FREETYPE_ROOT = None",
                 "FREETYPE_ROOT = ('#{Formula["freetype"].opt_prefix}/lib', " \
-                                 "'#{Formula["freetype"].opt_prefix}/include')"
+                "'#{Formula["freetype"].opt_prefix}/include')"
```

Comment:
Also awful, but we can `depends_on "pillow"` instead

Reply:
`volatility` uses python2 🙁

Reply:
Sad times. Maybe we can also build `pillow` for Python2...

Reply:
> Maybe we can also build pillow for Python2...
Oh, let's don't do it 🙂 

Also `volatility` 3 should support `python3`, and it seems the first version of it has been releasere (at least tagget): https://github.com/volatilityfoundation/volatility3, not sure about its status though and if we can replace current version with the new one